krony-app/dkron-api.tf

```diff
@@ -38,6 +42,35 @@ resource "kubernetes_service" "dkron" {
   }
 }
 
+resource "kubernetes_manifest" "dkron_metrics" {
```

Comment:
An alternative to the alpha provider can be to use a local helm chart: https://github.com/XenitAB/terraform-modules/tree/main/modules/kubernetes/external-dns/charts/external-dns-extras

Reply:
Well, the issue here turns out to be thst Terraform cannot at present apply CRDs and their CRs in the same run. That is a fairly serious limitation given the proliferation of CRDs in Kubernetes these days.